server/middleware/multer.js

```diff
@@ -0,0 +1,21 @@
+const multer = require('multer');
+const path = require('path');
+const DatauriParser = require("datauri/parser");
+
+const storage = multer.memoryStorage();
+const multerUploads = multer({ storage }).single("image");
+
+
+const parser = new DatauriParser();
+
+
+function convertBufferToString(req) {
+    let file64;
+     const extName = path.extname(req.file.originalname).toString();
```

Comment:
The formatting seems a bit off, make sure you configure prettier (for the BE) and set it to format on save 👍🏽